src/pages/setpassword/SetPasswordPage.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {
+    SafeAreaView, Text, View, Image,
+} from 'react-native';
+import styles from '../../styles/styles';
+import SignInPageLayout from '../signin/SignInPageLayout';
+import SetPasswordPageProps from './SetPasswordPageProps';
+import welcomeScreenshot from '../../../assets/images/welcome-screenshot.png';
+import withWindowDimensions from '../../components/withWindowDimensions';
+import SetPasswordForm from './SetPasswordForm';
+
+const SetPasswordPage = (props) => {
+    const welcomeText = (
+        <View style={props.isSmallScreenWidth ? [] : [styles.mb6, styles.mt6]}>
+            <Text style={[props.isSmallScreenWidth ? styles.textLabel : styles.textP, styles.textStrong, styles.mb1]}>
+                With Expensify.cash, chat and payments are the same thing.
```

Comment:
This is the 3rd use of this UI element so I think maybe it's time to create a reusable component out of this...? 🤔 

https://github.com/Expensify/Expensify.cash/blob/9b282f10de193d9a4115bc30f51e0dfdb4133050/src/pages/signin/LoginForm/LoginFormNarrow.js#L107-L109

https://github.com/Expensify/Expensify.cash/blob/9b282f10de193d9a4115bc30f51e0dfdb4133050/src/pages/signin/LoginForm/LoginFormWide.js#L98-L100